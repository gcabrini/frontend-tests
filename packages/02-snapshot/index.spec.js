import React from "react";
import { render } from "@testing-library/react";

import RoundedCard from "./";

describe("RoundedCard", () => {
  it("render properly", () => {
    const { container } = render(<RoundedCard>foo</RoundedCard>);

    expect(container.firstChild).toMatchSnapshot();
  });
});
