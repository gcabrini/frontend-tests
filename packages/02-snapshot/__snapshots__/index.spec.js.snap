// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RoundedCard render properly 1`] = `
<div
  class="RoundedCard"
>
  foo
</div>
`;
